assignment.js

```diff
@@ -105,9 +122,17 @@ console.log('Problem 3:')
 
 // Add your code below this line
 
+for(let i = 0; i < 100; i++) {
```

Comment:
🛠 Read the comments below and update your code accordingly

@rvaswani85 structurally, your logic is solid but the "order" of your `if` clauses are incorrect. For example number **15** is divisible by both 3 and 5, which means it should print out "fizzbuzz" but the order of your clauses will result in the program only printing "fizz". 

Also, your program is not currently printing out numbers that are not divisible by either 3 or 5.